Lib/test/test_minidom.py

```diff
@@ -1617,13 +1613,11 @@ def testEmptyXMLNSValue(self):
         self.confirm(doc2.namespaceURI == xml.dom.EMPTY_NAMESPACE)
 
     def testExceptionOnSpacesInXMLNSValue(self):
-        if pyexpat.version_info >= (2, 4, 5):
-            context = self.assertRaisesRegex(ExpatError, 'syntax error')
-        else:
-            context = self.assertRaisesRegex(ValueError, 'Unsupported syntax')
-
-        with context:
-            parseString('<element xmlns:abc="http:abc.com/de f g/hi/j k"><abc:foo /></element>')
+        with self.assertRaises((ValueError, ExpatError)):
+            parseString(
+                '<element xmlns:abc="http:abc.com/de f g/hi/j k">' +
```

Comment:
On a stylistic note: Because of the surrounding round braces, the trailing `+` can be dropped. Not a big deal, but could be dropped.

Reply:
I hope this is really not important for you: for me explicit is better than implicit and so when I want to have two string concatenated I prefer to make it obvious.

Reply:
I do not consider string concatenation without plus *im*plicit personally, but the topic was addressed, I'm good, not too important :+1: